src/parser.ts

```diff
@@ -8531,6 +8552,16 @@ export function parsePropertyDefinition(
 
     if (parser.token === Token.Arguments) report(parser, Errors.StrictEvalArguments);
 
+    const modifierFlags =
```

Comment:
I believe you copied some code from parseMethodDefinition.
There are few places that meriyah uses this hard to cipher binary, Kenny was very focused on performance, but he is retired now. I could not remember what it does (if I ever understood it before). Could you help to add few comments for what is does?
Thanks very much!

Reply:
Yes, you are right, I copied it from another place, namely from the `parseMethodDefinition`. Why I did it. It's because I noticed that Meriyah didn't throw the exception for the following js `class a { a() { super.a }}` and apparently it's because `parseMethodDefinition` changes the context. So, I did the same for `parsePropertyDefinition` and it started working for `class a { a = () => { super.a }}`.

Reply:
OK :-) Leave it then.